synthtool/gcp/templates/python_samples/README.md

```diff
@@ -0,0 +1,59 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `python -m synthtool`."
+
+## Python Sample Folder for {{ metadata['repo']['name_pretty'] }} 
+
+This directory contains samples for {{ metadata['repo']['name_pretty'] }}, which may be used as a refererence for how to use this product. {% if metadata['repo']['custom_content'] is defined %}
+{{ metadata['repo']['custom_content']}}{% endif %}
+
+## Samples
+{% if metadata['repo']['samples']|length %}
+Samples, quickstarts, and other documentation available for this product is available at <a href="{{ metadata['repo']['product_documentation'] }}">the product documentation.</a>
+{% for sample in range(metadata['repo']['samples']|length) %}
+
+### {{ metadata['repo']['samples'][sample]['name']}}
+
+{{ metadata['repo']['samples'][sample]['description']}}
+{% if metadata['repo']['samples'][sample]['runnable'] %}
+
+<a href="https://console.cloud.google.com/cloudshell/open?git_repo=https://github.com/{{ metadata['repo']['repo'] }}&page=editor&open_in_editor={{ metadata['repo']['samples'][sample]['file'] }}"><img alt="Open in Cloud Shell" src="http://gstatic.com/cloudssh/images/open-btn.png"> 
+</a>
+
+To run this sample:
+
+1. If this is your first time working with GCP products, you will need to set up [the Cloud SDK][cloud_sdk] or utilize [Google Cloud Shell][gcloud_shell]. This sample may [require authetication][authentication]{% if metadata['repo']['requires_billing'] %} and you will need to [enable billing][enable_billing]{% endif %}.
+
+1. Make a fork of this repo and clone the branch locally, then navigate to the sample directory you want to use.
+
+1. Install the dependencies needed to run the samples.
+        ```
```

Comment:
Can we remove the $ for easier copy pasting as well as have these snippets on their own lines similar to how Node does it?  https://github.com/googleapis/nodejs-bigtable#installing-the-client-library

Reply:
I thought I wasn't able to new lines of code blocks in but I checked up on how and fixed that up.